man/ronin-encode.1.md

```diff
@@ -86,6 +86,32 @@ Encodes each character of the given data into a variety of formats.
 `-h`, `--help`
   Print help information.
 
+## EXAMPLES
+
+Encode a string to base 16:
+
+        ronin encode --base16 --string {{string_name}}
```

Comment:
`{{string_name}}` should be replaced with an example double-quoted string. Maybe also the output of the command could also be shown:

```
$  ronin encode --base16 --string "hello world"
68656c6c6f20776f726c64
```